sql/core_clients_daily_v1.sql

```diff
@@ -82,6 +82,13 @@ WITH
     AND client_id != 'c0ffeec0-ffee-c0ff-eec0-ffeec0ffeec0'
   WINDOW
     w1 AS (
+    PARTITION BY
+      client_id,
+      submission_date_s3
+    ORDER BY
+      metadata.timestamp DESC
+    ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING),
```

Comment:
```suggestion
    ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING),
```

The result should be technically the same here since we're filtering out all but the first row in each partition, but it feels cleaner to me to specify `UNBOUNDED PRECEDING` rather than `CURRENT ROW`

Reply:
I agree, and less prone to any potential issues in the future